<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxElementsAmount">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxElementsAmount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonElementsAmount">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBoxInitialValues">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBoxElementNumber">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditInitialValue"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="comboBoxElementType">
        <item>
         <property name="text">
          <string>String</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Identifier String</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bit String</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonSetInitialData">
        <property name="text">
         <string>Confirm All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TESTING</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxTesting">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBoxOperandToTest"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEditExtraOperandToTest"/>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QTextBrowser" name="textBrowserTestingResultData"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxMethodToTest">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QComboBox" name="comboBoxClassToTest">
        <item>
         <property name="text">
          <string>String</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Identifier String</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bit String</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButtonTestMethod">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QTextBrowser" name="textBrowserTestingInitialData"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButtonExit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>by kndratova</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INITIALIZATION</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
